api/src/main/java/org/apache/cloudstack/query/QueryService.java

```diff
@@ -92,7 +92,7 @@
 
     static final ConfigKey<String> UserVMBlacklistedDetails = new ConfigKey<String>("Advanced", String.class,
             "user.vm.blacklisted.details", "rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag",
-            "Determines whether users can view certain VM settings", true);
+            "Determines whether users can view certain VM settings. When set to empty, default value used is: rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag.", true);
```

Comment:
Slight addition could be there optinally - "To allow all settings to be visible set this to `none`"?

Reply:
The main issue with setting to a fake value is that, what is the none is used as a valid detail (most likely that's never going to happen but for sanity-sake). Which is why I chose to simply state that behaviour.

Reply:
Sounds good.